calico/operations/upgrading/kubernetes-upgrade.mdx

```diff
@@ -120,6 +120,22 @@ the output and then re-running the command without --dry-run to commit to the ch
    kubectl apply --server-side --force-conflicts -f tigera-operator.yaml
    ```
 
+1. Optional: To enable the flow logs API and Calico Whisker (introduced in version 3.30) by applying the `Goldmane` and `Whisker` custom resources.
```

Comment:
Is this also needed in the other upgrade types?

Reply:
Other upgrade types are going in separately. Ditto with contra-indications.